frontend/src/app/admin/dashboard/ban/useBanManagement.ts

```
@@ -0,0 +1,116 @@
+    try {
+      const data = await getPlayers(query);
+      if (requestId !== latestRequestId.current) return;
+      setPlayers(data);
+    } catch {
+      if (requestId !== latestRequestId.current) return;
+      setToast({
+        message: "유저 목록을 불러올 수 없습니다.",
+        variant: "error",
+      });
+    } finally {
+      if (requestId !== latestRequestId.current) return;
```

Biome (2.4.7)

[error] Unsafe usage of 'return'.

(lint/correctness/noUnsafeFinally)